packages/client/src/app/components/card/CardList.vue

```diff
@@ -0,0 +1,26 @@
+<script setup>
+import CardItem from './CardItem.vue'
+
+const props = defineProps({
+  title: String,
+  emptyState: String,
+  items: Array
+})
+</script>
+
+<template>
+  <section class="mb-4">
+    <h3 class="fw-bold">{{ props.title }}</h3>
```

Comment:
What will the props.title be? I don't see anything showing up yet

Reply:
Yes, forgot to update the router after merging. Will fix it